api/client/login.go

```diff
@@ -26,7 +26,6 @@ func (cli *DockerCli) CmdLogin(args ...string) error {
 
 	flUser := cmd.String([]string{"u", "-username"}, "", "Username")
 	flPassword := cmd.String([]string{"p", "-password"}, "", "Password")
-	flEmail := cmd.String([]string{"e", "-email"}, "", "Email")
```

Comment:
Wonder if we should *remove* the flag, or only mark it deprecated for now (and simply ignore its value). Our deprecation policy is to mark options deprecated, but keep them around for two more releases, see for example: https://github.com/docker/docker/blob/master/docs/deprecated.md#hostconfig-at-api-container-start